org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/translator/A4Options.java

```diff
@@ -23,6 +23,8 @@
 /**
  * Mutable; this class encapsulates the customizable options of the
  * Alloy-to-Kodkod translator.
+ *
+ * @modified: Nuno Macedo // [HASLab] electrum-unbounded, electrum-decomposed
```

Comment:
adds options for the SMV backends (selection of solver) and the decomposed strategy (which mode and how many threads)